backend/src/migrate-api-keys.ts

```diff
@@ -0,0 +1,29 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+require('dotenv').config()
+require('module-alias/register')
+
+import sequelizeLoader from '@core/loaders/sequelize.loader'
+import { User } from '@core/models'
+import { ApiKey } from '@core/models/user/api-key'
+import { Op } from 'sequelize'
+void (async function main() {
+  await sequelizeLoader()
+  const users = await User.findAll({
+    where: { apiKeyHash: { [Op.not]: null } },
+  })
+  for (const u of users) {
```

Comment:
can be optimized if performance is a concern, but if it's a one time script to be ran after hours then no issues

Reply:
Actually this was intended, cos we don't have a lot of user records to begin with (can be queried within a single query without limit) and we're narrowing down to `api_key_hash not null`

Reply:
we have 46 active API users haha

Reply:
oh then defo no issue